win32/ioutil.c

```diff
@@ -1111,7 +1112,21 @@ PW32IO ssize_t php_win32_ioutil_readlink_w(const wchar_t *path, wchar_t *buf, si
 	ret = php_win32_ioutil_readlink_int(h, buf, buf_len);
 
 	if (ret < 0) {
-		/* BC */
+		/* BC - get a handle to the target (if path is a symbolic link) */
```

Comment:
Please have declarations first (for code consistency)

Reply:
Apologies, but I don't understand what you would like me to do here.

Reply:
Kalle suggests to move the newly inserted statements after the alreay existing declarations of `target`, `offset` and `target_len`. Having declarations before statements was mandatory for C89 compliance, and may still be preferable for current C standards (although that's likely to be controversial).

Reply:
Okay, thanks got it. I'll fix this.

Reply:
C99 flags has been merged actually ;)

Thanks.